engine/src/test/scala/org/scalarules/dsl/nl/grammar/AfrondingsTestBerekeningGlossary.scala

```diff
@@ -0,0 +1,19 @@
+package org.scalarules.dsl.nl.grammar
+
+import org.scalarules.finance.nl.Percentage
+import org.scalarules.utils.Glossary
+
+object AfrondingsTestBerekeningGlossary extends Glossary {
+
+  val aantal = defineFact[Integer]()
```

Comment:
I know, but leaving out the braces gives warnings and it's just a testglossary so I put them in anyway.